### ISSUE TITLE
Install recipe not working on Windows 10/Git Bash

### ISSUE THREAD

**Issue by bravo-kernel** [blitz-js/blitz#1651]
### What is the problem?

I am getting this error when trying to install a recipe (on Windows 10 using Git Bash). I have tried both chakra and tailwind.

```
✕ Error: Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default.
Read about how to prevent this error on https://github.com/vadimdemedes/ink/#israwmodesupported
```

- Error stack below mentions https://github.com/vadimdemedes/ink/#israwmodesupported.
- Might be a broader issue with Git Bash, see `Other`.

<details>
  <summary>Error Stack</summary>

```
$ blitz install tailwind
You are using alpha software - if you have any problems, please open an issue here:
      https://github.com/blitz-js/blitz/issues/new/choose

- Cloning GitHub repository for tailwind
- Installing package.json dependencies

  ERROR Raw mode is not supported on the current process.stdin, which Ink uses
       as input stream by default.
       Read about how to prevent this error on
       https://github.com/vadimdemedes/ink/#israwmodesupported

 - Read about how to prevent this error on
   https://github.com/vadimdemedes/ink/#israwmodesupported
 -App.handleSetRaw (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz
  ode             /node_modules/ink/src/components/App.tsx:139:11)
 - (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz/node_modules/in
  k/src/hooks/use-input.ts:123:3)
 -commitHookEffect (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz
  ist             /node_modules/react-reconciler/cjs/react-reconciler.developme
                  nt.js:12025:26)
 -commitPassiveHookE (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-bli
  ffects            tz/node_modules/react-reconciler/cjs/react-reconciler.devel
                    opment.js:12059:11)
 -Object.invokeGuardedCa (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js
  lbackImpl             -blitz/node_modules/react-reconciler/cjs/react-reconcil
                        er.development.js:11563:10)
 -invokeGuardedCal (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz
  back            /node_modules/react-reconciler/cjs/react-reconciler.developme
                  nt.js:11740:31)
 -flushPassiveEffec (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blit
  sImpl            z/node_modules/react-reconciler/cjs/react-reconciler.develop
                   ment.js:15390:7)
 -unstable_runWithPr (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-bli
  iority            tz/node_modules/scheduler/cjs/scheduler.development.js:697:
                    12)
 -runWithPrior (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz/nod
  ty          e_modules/react-reconciler/cjs/react-reconciler.development.js:18
              81:10)
 -flushPassiveEff (C:/Users/bravo/AppData/Local/Temp/blitz-recipe-blitz-js-blitz/
  cts            node_modules/react-reconciler/cjs/react-reconciler.development
                 .js:15359:12)

✕ Error: Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default.
Read about how to prevent this error on https://github.com/vadimdemedes/ink/#israwmodesupported

```

</details>

### Workaround

Use Git Bash "inside" Windows Terminal (and everything will work as expected).

### Steps to Reproduce

1. On Windows 10, start Git Bash
2. Run the following commands

```bash
blitz new tester
cd tester
blitz install chakra
```

### Versions

```
Windows 10 | win32-x64 | Node: v14.15.3

blitz: 0.27.1 (global)
blitz: 0.27.1 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz
    Memory: 8.30 GB / 15.94 GB
  Binaries:
    Node: 14.15.3 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - ~\.yarn\bin\yarn.CMD
    npm: 6.12.1 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: ~2.12 => 2.12.1
    @prisma/client: ~2.12 => 2.12.1
    blitz: 0.27.1 => 0.27.1
    react: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    react-dom: 0.0.0-experimental-4ead6b530 => 0.0.0-experimental-4ead6b530
    typescript: 4.1.3 => 4.1.3
```

### Other

This might not be limited to just installing recipes and could be a limitation in Git Bash. I just noticed I am unable to choose the form library when running `blitz new tester`. Cannot move through/select the options using keyboard. **Update:** same issue after upgrading Git Bash to latest version 2.30.0.

Refs #1334

**Comment by flybayer** [comment]
Hmm, that's not good! Thanks for reporting!

**Comment by bravo-kernel** [comment]
This looks resolved in current blitz version 0.29.7

Menu works, colors now show as well.
